### PR TITLE
Add an optional flag to remove the helm-docs version footer from the default template

### PR DESCRIPTION
**Pull request by gbprz**
This change adds a new flag named `skip-version-footer`. When set to `true`, the version footer is removed from the default README template. By default, the flag is set to `false` so the version footer is kept for backwards compatibility.

I had a similar issue as this comment https://github.com/norwoodj/helm-docs/issues/162#issuecomment-1819415642 where the default README template is great for the majority of our charts. For maintenance, it would be much easier for us to avoid copying README templates for each chart just to remove the version footer.

Fixes https://github.com/norwoodj/helm-docs/issues/162

### CODE REVIEW

File: pkg/document/generate.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func getOutputFile(chartDirectory string, dryRun bool) (*os.File, error) {
 	return f, err
 }
 
-func PrintDocumentation(chartDocumentationInfo helm.ChartDocumentationInfo, chartSearchRoot string, templateFiles []string, dryRun bool, helmDocsVersion string, badgeStyle string, dependencyValues []DependencyValues) {
+func PrintDocumentation(chartDocumentationInfo helm.ChartDocumentationInfo, chartSearchRoot string, templateFiles []string, dryRun bool, helmDocsVersion string, badgeStyle string, dependencyValues []DependencyValues, skipVersionFooter bool) {
```
Comment (gbprz) [inline pkg/document/generate.go:29]: I noticed this parameter list is getting long, I can create a struct to group similar parameters together either in this change or a follow up change if preferred.

For example, a PrintDocumentationParams for all params or a TemplateOpts struct that groups together settings like `badgeStyle`, `helmDocsVersion`, and `skipVersionFooter` that control the default template's behavior.

**Comment by mpluhar** [comment]
I guess this PR would supercede my one to update at least the documentation
https://github.com/norwoodj/helm-docs/pull/219

**Comment by norwoodj** [comment]
There have been requests to do this in the past and I still feel the same way: 

If you don't want the versionFooter in your template, you can use a README.md.gotmpl file to define the template that you want to use when rendering your README. The default template is what it is, if you don't like it, there is already a mechanism to define your own. I don't want to go down the road of adding a ton of options to augment the default template like this.

**Comment by weisscorp** [comment]
Man, you present your repository as ready for use in precommit hooks. But you say that I need to manually change some file. Now all the people who use your repository will go and change this file for themselves, instead of adding one flag. If you really insist, write instructions on how to change this file using precommit hooks.

**Comment by hstenzel** [comment]
> There have been requests to do this in the past and I still feel the same way:
> 
> If you don't want the versionFooter in your template, you can use a README.md.gotmpl file to define the template that you want to use when rendering your README. The default template is what it is, if you don't like it, there is already a mechanism to define your own. I don't want to go down the road of adding a ton of options to augment the default template like this.

Could you comment on the idea of keeping the footer but removing the version? I think that would solve the problem of unexpected/untimely changes that complicated workflows such as validation in CD and pre-commit.

Thanks!

**Comment by norwoodj** [comment]
I'll merge it. Thanks for the changes, I've been convinced.

**Pull request merged**